example/lib/main.dart

```diff
@@ -16,6 +15,8 @@ void main() {
 }
 
 class ChatScreen extends StoreWatcher {
+  final TextEditingController msgController = new TextEditingController();
+
   ChatScreen({Key key}) : super(key: key);
```

Comment:
nit: constructor first

see our style guide for more details on flutter style
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo